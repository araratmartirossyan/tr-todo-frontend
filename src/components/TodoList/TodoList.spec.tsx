import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { TodoList } from './index'

describe('Todo test', () => {
  it('should render component', () => {
    const component = shallow(
      <TodoList>
        <div></div>
        <div></div>
      </TodoList>
    )
    expect(toJson(component)).toMatchSnapshot()
  })
})
