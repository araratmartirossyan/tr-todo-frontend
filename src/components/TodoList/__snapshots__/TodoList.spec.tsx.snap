// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo test should render component 1`] = `
<styled.ul>
  <div />
  <div />
</styled.ul>
`;
